test/helpers/ussd_tasker.py

```diff
@@ -21,4 +26,4 @@ def inquire_balance(user: User):
 
     @staticmethod
     def fetch_user_exchange_rate(user: User):
-        pass
+        pass
```

Comment:
do you have some auto-formatting on that messes with the last newline?

Reply:
I have pylint running for my pages.